pandas/io/tests/test_packers.py

```diff
@@ -446,6 +446,37 @@ def test_sparse_panel(self):
                               check_panel_type=True)
 
 
+class TestCompression(TestPackers):
+
+    def setUp(self):
+        super(TestCompression, self).setUp()
+        data = {
+            'A': np.arange(1000, dtype=float),
+            'B': range(1000),
```

Comment:
use 'float64' as dtype for A, np.arange(....,dtype='int32') for B (otherwise the comparisons won't work on windows as the defaults for float/int are different).